machine/machine_core/machine.py

```diff
@@ -244,6 +244,10 @@ def get_admin_group(self):
         else:
             return "wheel"
 
+    def get_cockpit_container(self):
+        tag = "released" if "released" in self.execute("podman ps") else "latest"
```

Comment:
I don't understand this -- do we ever actually start the `:released` container in our tests? fedora-coreos.install uses this to back up the original container before refreshing it, but tests are always supposed to use `:latest`.

Reply:
Then I misunderstood the failure from https://logs.cockpit-project.org/logs/pull-2123-20210621-094713-4e63e07a-fedora-coreos-cockpit-project-cockpit/log.html#244-2

Will adjust, thanks for the pointer.

Reply:
NB I'm not saying it's wrong, I was just wondering where that comes from.

Reply:
So the install script of fedora-coreos tags the 'released' image as 'latest', so even when we see released in the podman ps it's actually the latest. I just realized it, i was quite confused.